common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -195,6 +196,7 @@ class Backend(
         val body = mapOf(
             "fetch_token" to purchaseToken,
             "product_ids" to receiptInfo.productIDs,
+            "product_plan_id" to receiptInfo.storeProduct?.googleProduct?.basePlanId,
```

Comment:
I think we should also send in the offerId together with this.
I don't like the fact that product_ids is a list, can we maybe do a `List<PlatformProductId>` instead where `PlatformProductId` has productId, basePlanId and offerId? Does that make sense?